phpcs.xml.dist

```diff
@@ -95,35 +89,67 @@
 	#############################################################################
 	-->
 
-
+	<!-- We still need to manually exclude the PHPCompatibility sniffs due to bug in phpcs
+		 (https://github.com/squizlabs/PHP_CodeSniffer/issues/2197)   -->
```

Comment:
I think it makes sense to hold off merging this PR until the bug is fixed. I don't think it's critical merging this one, and the extra workarounds add quite a bit of unnecessary complexity and confusion.

Reply:
I agree that there is a bit more overhead with manually silencing all the unnecessary warnings. The only worrying thing is that Greg Sherwood (maintainer of the phpcs) said that he would put this on 4.0.0 milestone, and unfortunately, the release date for v4 is not known

Reply:
I think we should hold off of that for now. I'm definitely inclined to migrate to using the `WPThemeReview` standard once ready, but things like the above indicate to me that it is not yet stable enough for wide adoption at this point.

Reply:
Yeah, it's intended for theme review on wordpress.org, so we added a compatibility check for PHP 5.2. In my projects, I'm using PHP 7.2 so I totally understand 🙂